react/src/components/KeypairResourcePolicyList.tsx

```diff
@@ -82,6 +81,8 @@ const KeypairResourcePolicyList: React.FC<KeypairResourcePolicyListProps> = (
             max_containers_per_session
             idle_timeout
             allowed_vfolder_hosts
+            max_pending_session_count @since(version: "24.03.4")
+            max_concurrent_sftp_sessions @since(version: "24.03.4")
```

Comment:
This also looks to be 23.03.3 in the schema.

https://github.com/lablup/backend.ai-webui/blob/a48214d3c7460710e09d05b431f97e8b33ccb04c/react/data/schema.graphql#L980-L988

Reply:
please refer https://github.com/lablup/backend.ai/issues/3413

Reply:
I'll leave comment. It occurs error when testing in 23.03.3